pages/linux/megadl.md

```diff
@@ -0,0 +1,17 @@
+# megadl
+
+> Download files from mega.nz.
+> Part of the megatools suite.
+> More information: <https://megatools.megous.com/>.
```

Comment:
I doesn't see any exact word of `megadl` in this link and its children links. Is there any specific website for this command?

Reply:
The official command is `megatools-dl` but, at least on my system, is automatically symlinked as `megadl`. `megadl` is (as far as I know) the command that's actually being used.

Reply:
Okay, I understand. I think this page should be rename to `megatools-dl.md` if which is the official name, and adding an alias page named `megadl.md` like [this](https://github.com/tldr-pages/tldr/blob/main/pages/common/vi.md).

Reply:
Also, this website is more suitable https://megatools.megous.com/man/megatools-dl.html for looking up usage .

Reply:
I think this sounds correct @blueskyson 👍🏻 We should try and use the official name, and if it is always symlinked, add an alias tldr page.

Reply:
@newtant Let us know if you need help to create this alias page.